src/test-support/mocks/src/hostname-generators/_/_overview.ts

```diff
@@ -0,0 +1,49 @@
+import type { HostnameGeneratorItem } from '@/app/hostname-generators/data'
+import type { EndpointDependencies, MockResponder } from '@/test-support'
+
+export default ({ fake, env }: EndpointDependencies): MockResponder => (req) => {
+  const { params } = req
+  const name = params.name as string
+  const [displayName = '', namespace = ''] = name.split('.')
+  const meshServiceTypeSelector = fake.kuma.meshServiceTypeSelector()
```

Comment:
Oh! I remember walking through this with you, I see that here in the end we didn't even need the CapitalCase form of the meshServiceType in the end 🎉 . Perfect!